source/fundamentals/crud/read-operations/retrieve.txt

```diff
@@ -0,0 +1,242 @@
+you want to find, then append the ``ToCursor`` or ``ToCursorAsync`` method as follows:
+
+.. tabs::
+
+   .. tab:: Asynchronous
+      :tabid: find-cursor-async
+
+      .. code-block:: csharp
+         :copyable: true
+
+         var restaurants = await _restaurantsCollection.Find(filter).ToCursorAsync();
+
+   .. tab:: Synchronous
+      :tabid: find-cursor-sync
+
+      .. code-block:: csharp
+         :copyable: true
+
+         var restaurants = _restaurantsCollection.Find(filter).ToCursor();
+
+Find Documents Using A List
+```````````````````````````
+
+If you are returning a small number of documents, or need your results returned
+as a ``List`` object, use the ``ToList`` or ``ToListAsync`` methods.
+
+To find multiple documents in a collection using a list, pass a query filter 
+to the ``Find()`` method that specifies the criteria of the documents you want 
+to find, then append the ``ToList`` or ``ToListAsync`` method as follows:
+
+.. tabs::
+
+   .. tab:: Asynchronous
+      :tabid: find-list-async
+
+      .. code-block:: csharp
+
+         var restaurants = await _restaurantsCollection.Find(filter).ToList();
+
+   .. tab:: Synchronous
+      :tabid: find-list-sync
+
+      .. code-block:: csharp
+
+         var restaurants = _restaurantsCollection.Find(filter).ToListAsync();
+
+To see a full example of using ``Find()`` to find multiple documents, see
+:ref:`csharp-retrieve-additional-information`.
+
+Find All Documents
+~~~~~~~~~~~~~~~~~~
+
+To find all documents in a collection, pass an empty
+``BsonDocument`` as a query filter. Then append the ``ToList`` or ``ToListAsync``
+method to return the documents as a ``List`` object, or the ``ToCursor`` or ``ToCursorAsync``
+method to return the documents as a ``Cursor``.
+
+To see a fully runnable example of using ``Find()`` to find all documents, see 
+:ref:`csharp-retrieve-additional-information`.
+
+Modify Find Behavior
+~~~~~~~~~~~~~~~~~~~~
+
+You can modify the behavior of ``Find()`` by passing
+in ``FindOptions``. If you don't specify any options, the driver uses the 
+default values for each option.
+
+You can configure the commonly used options with the following methods:
+
+.. list-table::
+   :widths: 30 70
+   :header-rows: 1
+
+   * - Method
+     - Description
+
+   * - ``BatchSize`` 
+     - | Gets or sets the number of documents to return per batch.
+
+   * - ``Collation`` 
+     - | Sets the collation options. 
+
+   * - ``Comment`` 
+     - | Sets the comment to the query.
+
+   * - ``Hint`` 
+     - | Sets the hint for which index to use.
+       | Default: ``0``
+
+   * - ``MaxTime`` 
+     - | Sets the maximum execution time on the server for this operation.
+
+To see a full list of available options, see 
+`FindOptions Properties <{+api-root+}/html/Properties_T_MongoDB_Driver_FindOptions.htm>`__.
+
+Example
+~~~~~~~
+
+The following example passes a query filter and ``FindOptions`` to the ``Find()``
+method which performs the following actions:
```

Comment:
Issue: I believe this description doesn't put the value proposition (configure/modify ``Find`` behavior) of the example at the beginning of the sentence.

```suggestion
The following example configures the behavior of the ``Find()`` method with a ``FindOptions`` object.
The example performs the following actions:
```